pkg/utils/labels_annotations.go

```diff
@@ -361,3 +365,24 @@ func MergeMap(receiver, giver map[string]string) {
 		receiver[key] = value
 	}
 }
+
+// GetInstanceRole tries to fetch the ClusterRoleLabelName andClusterInstanceRoleLabelName value from a given labels map
+func GetInstanceRole(labels map[string]string) (string, bool) {
+	if value := labels[ClusterRoleLabelName]; value != "" {
```

Comment:
shouldn't we swap the two checks here? also please add a TODO that we will remove this from 1.22.1 or later.

Reply:
It makes no difference, given that the contain virtually the same value.

A TODO is not needed here IMO, because once we remove the old ClusterRoleLabelName, the project won't compile because the referenced constant is used here